bpf/kmesh/workload/include/authz.h

```diff
@@ -0,0 +1,208 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause) */
+/* Copyright Authors of Kmesh */
+
+#ifndef __AUTHZ_H__
+#define __AUTHZ_H__
+
+#include "workload_common.h"
+#include "bpf_log.h"
+#include "workloadapi/security/authorization.pb-c.h"
+
+#define AUTH_ALLOW   0
+#define AUTH_DENY    1
+#define UNMATCHED    0
+#define MATCHED      1
+#define UNSUPPORTED  2
+#define TYPE_SRCIP   (1)
+#define TYPE_DSTIP   (1 << 1)
+#define TYPE_DSTPORT (1 << 2)
+
+struct {
+    __uint(type, BPF_MAP_TYPE_HASH);
+    __uint(key_size, AUTHZ_NAME_MAX_LEN);
+    __uint(value_size, sizeof(Istio__Security__Authorization));
+    __uint(map_flags, BPF_F_NO_PREALLOC);
+    __uint(max_entries, MAP_SIZE_OF_AUTH);
```

Comment:
This size no need to be so large, iiuc this is to store authz policy for the pods within a node

Reply:
fix